src/Components/Book/Book.js

```diff
@@ -0,0 +1,13 @@
+import './Book.css'
+
+const Book = ({ img, title, author, id }) => {
+  return (
+    <div className='book-card'>
+      <img src={img}></img>
+      <h2>{title}</h2>
+      <h3>By {author}</h3>
+    </div>
```

Comment:
Will probably make the entire book card a link to the info page for that book - Will need dynamic URL that includes the book's ISBN/id. Or it could just be the image is the link, but not sure if that is as UX/UI friendly.